code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -146,6 +146,24 @@
 		var/datum/action/A=X
 		A.UpdateButtonIcon()
 
+/obj/item/clothing/head/helmet/space/plasmaman/cmo
+	name = "CMO plasma envirosuit helmet"
+	desc = "A plasmaman containment helmet designed for the chief medical officer."
```

Comment:
Inconsistent capitalisation with 'chief medical officer' then 'Research Director'. Can you capitalise all the job titles pls?

Reply:
Sure, I will go ahead and do that now